src/EditorFeatures/VisualBasicTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.vb

```diff
@@ -2421,6 +2421,19 @@ End Class",
 End Class")
         End Function
 
+        <WorkItem(19498, "https://github.com/dotnet/roslyn/issues/19498")>
+        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsSimplifyTypeNames)>
+        Public Async Function TestMyClassShouldNotBeRemoved() As Task
+            Await TestMissingInRegularAndScriptAsync(
+"Class SomeType
+    Overridable Sub Test()
```

Comment:
I think covering all the cases is the way to go unless it causes issues somewhere.
Enough for today, will continue tomorrow and hopefully be more effective now that I have at least some grasp of the code 😀